docs/src/00-doc/05-decisions-and-ADRs/adr/0012-remove-widths.stories.mdx

```diff
@@ -0,0 +1,38 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="Documentation/Decisions and ADRs/ADRs/0012) Remove predefined widths" />
+
+# 12) Remove predefined component widths
+
+Date: 2020-11-10
+
+## Status
+
+Proposed
+
+## Context
+
+Originally, block-level system components – those which size is not defined by their content (e.g. input field) – were designed and implemented following a set of predefined default widths based on the system's spacing unit (the most common size being 256px, for example). 
+These components were also designed and implemented to take a "full-width" value as a prop, intended to make them responsive and adjustable to their context via their containers.
+
+While these predefined widths were purely provided as orientation, we inferred that they might be unused after all due to the variability of contexts in which components can be implemented. 
+Making components full-width by default would simplyfy things: making them more versatile, responsive and adjustable to the layout of their application. 
+
+## Considered actions
+
+### 1. Keep predefined widths, but make "full-width" the default value.
+
+This conservative approach would make block-level components "full-width" by default via prop. Predefined width values are still available to be applied.
+
+### 2. Get rid of widths entirely and make components' sizes to be limited to their layout element.
+
+This involves removing all predefined widths and width props. As long as they're block-level, components will take the width of their container.
+
+## Decision
+
+Remove width props and make components' widths 100% by default, so their final width is defined in context, by their layout elements. 
+Containers will still be able to take predefined system widths if suitable, using dimension tokens.
```

Comment:
I'm not sure what this means, does it just mean that implementers will still be able to use dimension tokens to size the containers in their own application?

Reply:
Yeah, it's not very well phrased, but that's the meaning. Let me rewrite that. Otherwise, is this worth mentioning? Thanks, @itamargiv 🙏

Reply:
Yeah, I think it's important to mention